File: Data Structures & Algorithms/counting-bits/submission-0.java
class Solution {
    public int[] countBits(int n) {
        int[] arr = new int[n+1];
        for(int i=0;i<n+1;i++){
            arr[i] = count1(i);
        }
        return arr;
    }
    public int count1(int n){
        int res = 0;
        while(n!=0){
            n&=(n-1);
            res++;
        }
        return res;
    }
}

File: Data Structures & Algorithms/merge-intervals/submission-0.py
class Solution:
    def merge(self, intervals: List[List[int]]) -> List[List[int]]:
        intervals.sort()
        res=[intervals[0]]
        for first,second in intervals[1:]:
            last_end=res[-1][1]
            if first<=last_end:
                res[-1][1]=max(last_end,second)
            else:
                res.append([first,second])
        return res

